cdhweb/blog/models.py

```diff
@@ -171,3 +243,15 @@ class BlogLinkPage(LinkPage):
     # NOTE the only allowed child page type is a BlogPost; this is so that
     # Events made in the admin automatically are created here.
     subpage_types = [BlogPost]
+
+
+class BlogLandingPage(Page, StandardHeroMixinNoImage):
```

Comment:
We should probably switch the order of these parent classes -- methods (and other things, i imagine) are resolved left-to-right, so anything `Page` implements will override anything `StandardHeroMixinNoImage` implements.

For example, if `StandardHeroMixin` ever overrode `get_context` it wouldn't come through here